### PR TITLE
frontend/project_settingspy: add enabled plugins list to settings

### LINKED ISSUE #525: Plugins settings missing from web UI for project/group admins
It isn't possible to enable/disable plugins when using the self-service web UI. The only option to change plugin settings for the project is to use admin interface.

### PR DESCRIPTION
**Pull request by chaws**
Closes #525

### CODE REVIEW

File: squad/frontend/project_settings.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 class ProjectForm(forms.ModelForm):
     class Meta:
         model = Project
-        fields = ['name', 'is_public', 'is_archived', 'description']
+        fields = ['name', 'is_public', 'is_archived', 'description', 'enabled_plugins_list']
```
Comment (mwasilew) [inline squad/frontend/project_settings.py]: could you also add timeouts?

Comment (chaws) [reply squad/frontend/project_settings.py]: "notification timeout" and "wait for notification"? I also thought about "data rentention", what do you think?

Comment (mwasilew) [reply squad/frontend/project_settings.py]: makes sense.

Comment (chaws) [reply squad/frontend/project_settings.py]: added

**Pull request merged**